redbot/cogs/audio/core/events/dpy.py

```diff
@@ -287,7 +390,7 @@ async def cog_before_invoke(self, ctx: commands.Context) -> None:
                         await sent.add_reaction("\N{CROSS MARK}")
                     raise commands.CheckFailure
                 else:
-                    if message.content.strip() != token:
+                    if compare_digest(message.content.strip(), token):
```

Comment:
```suggestion
                    if not compare_digest(message.content.strip(), token):
```
If the user types the correct captcha it will error, but if they type the wrong one then the command goes through regardless.

Found by testing `[p]llset reset` for #5651.

![Screenshot from 2022-04-09 14-52-08](https://user-images.githubusercontent.com/40699625/162576388-a849386b-ac70-4239-b85c-224a7343cd08.png)
![Screenshot from 2022-04-09 14-52-17](https://user-images.githubusercontent.com/40699625/162576393-e557e7d1-aabe-4518-8b2f-cb39e47bea19.png)

Reply:
Fixed by [`6f74c48` (#5649)](https://github.com/Cog-Creators/Red-DiscordBot/pull/5649/commits/6f74c486f4e96b1827617e2fb142a66121f86893)